WWDCFamily/Source/Map/Controllers/MapViewController.swift

```diff
@@ -1,18 +1,18 @@
 import UIKit
 import MapKit
 
-final class MapViewController: UIViewController, StoryboardInstance, RootChildViewController {
+final class MapViewController: UIViewController, RootChildViewController {
 
-    static let storyboardName: String = "Core"
+    private let datasource = MapDataSource()
 
-    @IBOutlet private weak var mapView: MKMapView! {
-        didSet {
-            mapView.setRegion(datasource.sfRegion, animated: true)
-            mapView.showsUserLocation = true
-        }
+    override func loadView() {
+        let view = UIView.instanceFromNib() as MapView
```

Comment:
What are the advantages to having an xib for the view rather then the controller itself?

It seems if we go this route we won't be able to wire up IBActions (to the vc) and we will require a specific delegate call back to the vc for every UI event.